.mise/tasks.toml

```diff
@@ -0,0 +1,75 @@
+# Copyright © SixtyFPS GmbH <info@slint.dev>
+# SPDX-License-Identifier: GPL-3.0-only OR LicenseRef-Slint-Royalty-free-2.0 OR LicenseRef-Slint-Software-3.0
+
+# Use https://mise.jdx.dev/schema/mise-task.json for validation.
+
+### fixes
+
+["fix:cpp:format"]
+description = "Run clang format fix on all C++ files"
+run = '''fd '.*\.(h|H|hpp|hxx|h\+\+|c|C|cpp|cxx|c\+\+)$' -0 | xargs -0 clang-format -i'''
+tools = { "pipx:clang-format" = "latest" }
```

Comment:
Give that the output of clang format changes, wouldn't it perhaps make sense to pin a version?

Reply:
Hmmm... right now we use "whatever is in Ubuntu 24.04" -- which is version 10.5 from what I can tell. I can set it to that, but that is rather far behind IMHO. Maybe fix it to version "20"?

Reply:
Actually this PR contains all the reformatting the clang-format version jump from `10.0.5` to `20.1.0`. They are just a few and they all look sane to me, so I would not mind just sticking it out with `latest` clang-format and see how it works out.

But then the other argument could be made, too: Why update when it won't change anyway?

Reply:
I'll leave this as is for now.